actions/src/dispatcher.js

```diff
@@ -27,9 +27,9 @@ module.exports = {
     const checkEvent = `${event}_${action}`;
     const repoName = context.payload.repository.name.toLowerCase();
     const checksWhitelist = constants.getChecksWhitelist();
-    if (checksWhitelist.hasOwnProperty(repoName)) {
+    if (Object.prototype.hasOwnProperty.call(checksWhitelist, repoName)) {
```

Comment:
Why is this changed?

Reply:
As per the eslint, if we call `hasOwnProperty` from any object directly, this might lead to unsafe way. So, using `Object.prototype` is a safe way to prevent such errors or bug might occur in future.  

It gives the following lint error:

```
30:25  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
32:18  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
```

Reply:
Got it. 
Thanks for the explanation.